packages/battery/battery/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: battery
 description: Flutter plugin for accessing information about the battery state
   (full, charging, discharging) on Android and iOS.
 homepage: https://github.com/flutter/plugins/tree/master/packages/battery
```

Comment:
The homepage needs to be changed as well to: battery/battery

Reply:
Changes are done now.